unittest/test_codegen.cpp

```diff
@@ -113,11 +120,26 @@ const boost::shared_ptr<crocoddyl::ActionModelAbstractTpl<Scalar> > build_arm_ac
   // Create a cost model per the running and terminal action model.
   boost::shared_ptr<CostModelSum> runningCostModel = boost::make_shared<CostModelSum>(state);
 
+  VectorXs lowlim = (model.lowerPositionLimit);
+  VectorXs uplim = (model.upperPositionLimit);
+  VectorXs xlb(model.nq + model.nv), xub(model.nq + model.nv);
+  xlb << lowlim, -VectorXs::Ones(model.nv);
+  xub << uplim, VectorXs::Ones(model.nv);
+
+  // xlb.tail(model.nv) *= Scalar(-1) * std::numeric_limits<Scalar>::max();
```

Comment:
Is there a reason to keep this commented lines?

Reply:
For now, this is an improvement I need to do in pinocchio. This might just be a reminder.